src/index.ts

```diff
@@ -420,7 +420,7 @@ const transform = async ({
   const functionsFile = join(
     tmpdir(),
     `functions-${Math.random().toString(36).slice(2)}.js`
-  );
+  ).replaceAll("\\", "/");
```

Comment:
For this and all the other `replaceAll` calls here I think this PR should wait for https://github.com/cloudflare/next-on-pages/pull/86 to get merged first and reuse the `normalizePath` that is being defined there

Reply:
sure

Reply:
@LOLBRUHNICE the other PR's got merged, feel free to rebase and use `normalizePath` 😃

Reply:
Now I changed all `replaceAll()` to `normalizePath()`, and since we have `normalizePath()`, I have removed the `tsconfig.json` as well